google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/stub/PublisherStubSettings.java

```diff
@@ -475,12 +475,14 @@ public PublisherStub createStub() throws IOException {
 
   /** Returns the default service endpoint. */
   public static String getDefaultEndpoint() {
-    return "pubsub.googleapis.com:443";
+    //    return "pubsub.googleapis.com:443";
+    return "staging-pubsub.sandbox.googleapis.com:443";
```

Comment:
I am sure you plan to already, but please make sure to revert this. Also, please don't submit these changes and keep them local only, since they live in the history.